bin/sh/install-boost.sh

```diff
@@ -10,7 +10,7 @@ set -e
 
 if [ ! -d "$BOOST_ROOT/lib" ]
 then
-  wget $BOOST_URL -O /tmp/boost.tar.gz
+  wget --no-check-certificate $BOOST_URL -O /tmp/boost.tar.gz
```

Comment:
Yes, I can work on that separately.  

@MarkusTeufelberger those tar.gz are effectively empty.  It doesn't seem to archive the submodules.